doc/news/changes/minor/20230801Munch_2

```diff
@@ -0,0 +1,4 @@
+New: The new function Triangulation::as_dof_handler_level_iterator()
```

Comment:
The $SUBJECT of the PR and this line confused me, because you're putting the function into `CellAccessor`. Would you mind fixing the changelog entry?

Separately, why not put this into `TriaAccessor`?